ArduPlane/mode.h

```diff
@@ -11,6 +11,12 @@
 #include <AP_Mission/AP_Mission.h>
 #include "pullup.h"
 
+#ifndef AP_QUICKTUNE_ENABLED
+#define AP_QUICKTUNE_ENABLED HAL_QUADPLANE_ENABLED
+#endif
+
+#include <AP_Quicktune/AP_Quicktune.h>
```

Comment:
Slightly odd place for the include.

Reply:
it is there so we can override the define in AP_QuickTune.h based on quadplane enable. I'm sure there are other ways to do it, but this works fine